docs/development/codeguide.rst

```diff
@@ -295,17 +297,18 @@ Including C Code
   provided the needed functionality. When C extensions are used, the Python
   interface must meet the aforementioned Python interface guidelines.
 
-* The use of Cython_ is strongly recommended for C extensions, as per the
-  example in the template package. Cython_ extensions should store ``.pyx``
-  files in the source code repository, but they should be compiled to ``.c``
-  files that are updated in the repository when important changes are made to
-  the ``.pyx`` file.
+* The use of Cython_ is strongly recommended for C extensions. Cython_
+  extensions should store ``.pyx`` files in the source code repository,
+  but not the generated ``.c`` files.
 
 * If a C extension has a dependency on an external C library, the source code
   for the library should be bundled with the Astropy core, provided the
   license for the C library is compatible with the Astropy license.
   Additionally, the package must be compatible with using a system-installed
-  library in place of the library included in Astropy.
+  library in place of the library included in Astropy, and a user installing
+  the package should be able to opt-in to using the system version using
+  a ``ASTROPY_USE_SYSTEM_???`` environment variable, where ``???`` is the name
```

Comment:
Can you please provide examples of what `???` can possibly be in real life?

Reply:
Done